blueprints/node-webkit/index.js

```diff
@@ -50,5 +51,22 @@ module.exports = {
         return writeFile(packageJsonPath, JSON.stringify(json, null, '  '));
       }
     });
+  },
+  addQUnitLogger: function(){
+    var testHelperFile = path.resolve(this.project.root, 'tests/test-helper.js');
+    var _this = this;
+    
+    console.log('modifying tests/test-helper.js to require vendor/qunit-logger');
```

Comment:
In that case, yes, you could use the `this.ui.writeLine()` like I do [here](https://github.com/brzpegasus/ember-cli-node-webkit/blob/v0.2.0/blueprints/node-webkit/index.js#L47). Check out Ember CLI's [architecture](https://github.com/ember-cli/ember-cli/blob/master/ARCHITECTURE.md#ui) to read more on `ui` vs. `console.log`.